packages/shared/src/hooks/squads/useSquad.ts

```diff
@@ -41,3 +41,24 @@ export const useSquad = ({ handle }: UseSquadProps): UseSquad => {
       : !!getApiError(error as ApiErrorResult, ApiError.Forbidden),
   };
 };
+
+type UseSquads = {
+  squads: Squad[];
+  isLoading: boolean;
+};
+
+export const useSquads = (): UseSquads => {
+  const { isFetched: isBootFetched, user } = useContext(AuthContext);
```

Comment:
Why do we care if boot is fetched?
Basically if you have a user you can go right?